modules/nw-networkpolicy-about.adoc

```diff
@@ -7,7 +7,6 @@
 = About network policy
 
 In a cluster using a network plugin that supports Kubernetes network policy, network isolation is controlled entirely by `NetworkPolicy` objects.
-In {product-title} {product-version}, OpenShift SDN supports using network policy in its default network isolation mode.
```

Comment:
That first sentence is still implicitly talking about old openshift-sdn multitenant mode, and is not true in general. (Isolation is also affected by AdminNetworkPolicy, EgressFirewall, etc.)

You can probably remove both sentences here, though probably you should move the WARNING to later in the file in that case